app/config.py

```diff
@@ -8,7 +8,8 @@ class DefaultConfig(object):
     # standard: no trailing slashes anywhere in variables.
     # with protocol
     URL_BASE = "https://anagora.org"
-    API_BASE = "http://localhost:3000"
+    API_BASE = "https://api.anagora.org:3000"
```

Comment:
I think we want just https://api.anagora.org, I'll have a reverse proxy through nginx to serve this at 443 in this virtual host